tests/test_compile_error.rs

```diff
@@ -1,13 +1,15 @@
 #![cfg(feature = "macros")]
 
 #[rustversion::stable]
+#[cfg_attr(target_arch = "wasm32", ignore)]
```

Comment:
Similarly, for this and others in the file...

```suggestion
#[cfg_attr(target_arch = "wasm32", ignore = "cannot invoke rustc for UI tests with wasm")]
```